bin/ch/WScriptJsrt.cpp

```diff
@@ -2039,6 +2005,17 @@ HRESULT WScriptJsrt::ModuleMessage::Call(LPCSTR fileName)
     return errorCode;
 }
 
+JsErrorCode WScriptJsrt::ReportModuleCompletionCallback(JsModuleRecord module, JsValueRef exception)
```

Comment:
Callback used when a module completes - returns the exception if the completion was not a normal completion.

This had to be added to support asynchronous module completion - note see below if the callback is not set engine will fallback to direct reporting for modules without Top Level Await to avoid any compatability break.